### PR TITLE
MAT-7933 updating stratification association in test cases when it is changed in groups

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## MADiE PR

Jira Ticket: [MAT-7933](https://jira.cms.gov/browse/MAT-7933)
(Optional) Related Tickets:

### Summary
updating stratification association in test cases when it is changed in groups

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/cms/gov/madie/measure/config/UpdateQICoreTestCaseStratificationsChangeUnit.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    log.debug("Entering updateQICoreTestCaseStratifications()");
+    List<Measure> measureList = measureRepository.findAllByModel(ModelType.QI_CORE.getValue());
+    log.info("found QI Core measures: {}", measureList.size());
+
+    measureList.forEach(
+        measure -> {
+          if (!measure.isActive()
+              || CollectionUtils.isEmpty(measure.getGroups())
+              || CollectionUtils.isEmpty(measure.getTestCases())) {
+            return;
```
Comment (adongare) [inline src/main/java/cms/gov/madie/measure/config/UpdateQICoreTestCaseStratificationsChangeUnit.java:37]: I thought foreach can't be broken with break or return statement?  does it really break?

Comment (sb-prateekkeerthi) [reply src/main/java/cms/gov/madie/measure/config/UpdateQICoreTestCaseStratificationsChangeUnit.java:37]: Yes, Ashok it does break out of that entry if you use return. I have tested it locally

**Review by adongare — APPROVED** [review]
